randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -556,21 +557,29 @@ Extra - asset_id: 85
   > Door to Hopper Energy Tank Room
       All of the following:
           Any of the following:
+              # Open the gate
               Wave Beam
-              All of the following:
-                  Missile or Super Missile
-                  Any of the following:
-                      Gate Glitch (Expert)
-                      Hi-Jump Boots and Gate Glitch (Advanced)
+              # Gate Glitch
+              Super Missile ≥ 3 and Gate Glitch (Expert)
+              # Gate Glitch
+              Hi-Jump Boots and Super Missile ≥ 3 and Gate Glitch (Advanced)
           Any of the following:
-              Grapple Beam or Space Jump or Wall Jumps (Beginner)
-              Morph Ball Bombs and Morph Ball and Vertical Infinite Bomb Jump (Intermediate)
+              # A way to get up
+              Grapple Beam or Space Jump or Wall Jumps (Intermediate)
+              # IBJ
+              Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+          # Defeat the Hoppers
+          Combat (Intermediate) or Normal Damage ≥ 150
 
 > Door to Hopper Energy Tank Room; Heals? False
   * Layers: default
   * Normal Door to Hopper Energy Tank Room/Door to Pink Brinstar Hopper Room
   > Door to Big Pink
-      Space Jump
+      All of the following:
+          # Get up
+          Space Jump or Wall Jumps (Intermediate)
+          # Defeat the Hoppers
+          Combat (Intermediate) or Normal Damage ≥ 150
```

Comment:
Its my understanding that 150 normal damage would be met with Samus' starting health of 99 + 1etank = 199 damage. Even if the preset multiplies the damage requirements 199*1.5 = 298.5 which is about Staring + 2 etanks.

Reply:
thats for 150, yes. but first the starter preset multiplies every damage value by 1.5 (aka 225), and second, because you get there and back you have to double that amount (450=

Reply:
If you assume beginner settings, then the player would also have Wave Beam which makes it require even less. I'm fine turning the energy down to 75. 1.5(75+75) = 225 < Starting + 2 etanks

Reply:
would bump that a little higher to 90. but seems fine otherwise.